ansible_collections/arista/avd/plugins/action/cv_workflow.py

```diff
@@ -44,44 +44,46 @@
+            "structured_config_suffix": {"type": "str", "default": "yml"},
+            "device_list": {"type": "list", "elements": "str", "required": True},
+            "strict_tags": {"type": "bool", "required": False, "default": False},
+            "skip_missing_devices": {"type": "bool", "required": False, "default": False},
+            "configlet_name_template": {"type": "str", "default": "AVD-${hostname}"},
+            "cv_servers": {"type": "list", "elements": "str", "required": True},
+            "cv_token": {"type": "str", "secret": True, "required": True},
+            "cv_verify_certs": {"type": "bool", "default": True},
+            "workspace": {
+                "type": "dict",
+                "options": {
+                    "name": {"type": "str", "required": False},
+                    "description": {"type": "str", "required": False},
+                    "id": {"type": "str", "required": False},
+                    "requested_state": {"type": "str", "default": "built", "choices": ["pending", "built", "submitted", "abandoned", "deleted"]},
+                    "force": {"type": "bool", "default": False},
+                },
             },
-        },
-        "change_control": {
-            "type": "dict",
-            "options": {
-                "name": {"type": "str", "required": False},
-                "description": {"type": "str", "required": False},
-                "requested_state": {"type": "str", "default": "pending approval", "choices": ["pending approval", "approved", "running", "completed"]},
+            "change_control": {
+                "type": "dict",
+                "options": {
+                    "name": {"type": "str", "required": False},
+                    "description": {"type": "str", "required": False},
+                    "requested_state": {"type": "str", "default": "pending approval", "choices": ["pending approval", "approved", "running", "completed"]},
+                },
             },
-        },
-        "timeouts": {
-            "type": "dict",
-            "options": {
-                "workspace_build_timeout": {"type": "float", "default": CVTimeOuts.workspace_build_timeout},
-                "change_control_creation_timeout": {"type": "float", "default": CVTimeOuts.change_control_creation_timeout},
+            "timeouts": {
+                "type": "dict",
+                "options": {
+                    "workspace_build_timeout": {"type": "float", "default": CVTimeOuts.workspace_build_timeout if HAS_PYAVD else 300.0},
+                    "change_control_creation_timeout": {"type": "float", "default": CVTimeOuts.change_control_creation_timeout if HAS_PYAVD else 300.0},
+                },
+                "return_details": {"type": "bool", "required": False, "default": False},
             },
         },
-        "return_details": {"type": "bool", "required": False, "default": False},
-    }
+    )
```

Comment:
Why is this changing all over? Looks like an extra parenthesis by mistake.

Reply:
let me recheck this - it was causing issue on merge - there is an extra HAS_PYAVD in one of the two files